### ISSUE TITLE
ALLOWEDIPS environment variable has no effect for newly created peers

### ISSUE THREAD

**Issue by P-Takenaka** [linuxserver/docker-wireguard#88]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior

The ALLOWEDIPS environment variable should determine the allowed ip range config field of new peers.

## Current Behavior

The allowed ip range for new peers always stays at the default "AllowedIPs = 0.0.0.0/0, ::/0".

## Steps to Reproduce

1. Set ALLOWEDIPS to some ip address range
2. Create new peer config files
3. New peer config files will have default "AllowedIPs = 0.0.0.0/0, ::/0".

## Environment
**OS:** Arch Linux 4.19.0-13-amd64
**CPU architecture:** x86_64
**How docker service was installed:** pacman

## Command used to create docker container (run/create/compose/screenshot)

```
services:
  wireguard:
    image: linuxserver/wireguard:v1.0.20200827-ls7
    cap_add:
      - NET_ADMIN
      - SYS_MODULE
    environment:
      - PUID=1000
      - PGID=1000
      - TZ='Germany/Frankfurt'
      - SERVERURL=example.com #optional
      - SERVERPORT=65303 #optional
      - PEERS=3 #optional
      - PEERDNS=192.168.1.10 #optional
      - INTERNAL_SUBNET=10.13.13.0 #optional
      - ALLOWEDIPS=192.168.1.0/24
    volumes:
      - /lib/modules:/lib/modules
    ports:
      - 65303:51820/udp
    sysctls:
      - net.ipv4.conf.all.src_valid_mark=1
    restart: unless-stopped
```

**Comment by aptalca** [comment]
It works here:
```
aptalca@jenkins:~$ docker run -d --rm --name test --cap-add=NET_ADMIN --cap-add=SYS_MODULE -e PEERS=1 -e ALLOWEDIPS=1.1.1.1 -v /lib/modules:/lib/modules ghcr.io/linuxserver/wireguard
910725ac08e08e37ff3b73448e2fdd03dbd99efe0b9cc3a8e31d9387753d0e04
aptalca@jenkins:~$ docker exec -it test cat /config/peer1/peer1.conf
[Interface]
Address = 10.13.13.2
PrivateKey = UOiUYDPYATWOrtFwEa8dPFw90Hh9ptwWM+C4wgmVqGE=
ListenPort = 51820
DNS = 10.13.13.1

[Peer]
PublicKey = dPJ4I5e4Ht19PX2dUePPcgZb+iU+BsNHuYEcphVxpDA=
Endpoint = 71.179.132.164:51820
AllowedIPs = 1.1.1.1
```

**Comment by P-Takenaka** [comment]
Thank you for testing it. I checked again and found my issue, the peer template file. In between tests, I only deleted the peer folders and wg0.conf to force config regeneration. However, the peer template also has to be deleted so that newly created peers will take the current environment variables.

What was confusing for me was that wireguard logs the current value in the environment:
```
wireguard           | **** AllowedIPs for peers 192.168.1.0/24 ****
```
even though the actual peer template will have a different value.

While the issue can easily be solved, I feel there should at least be a clarification in the docs that the peer template file has to be deleted manually in order for changes to peer environment variables to take effect.
